janusgraph-core/src/main/java/org/janusgraph/core/JanusGraph.java

```diff
@@ -28,15 +28,8 @@
  * @see JanusGraphTransaction
  */
 @Graph.OptIn(Graph.OptIn.SUITE_STRUCTURE_STANDARD)
-@Graph.OptIn(Graph.OptIn.SUITE_STRUCTURE_PERFORMANCE)
 @Graph.OptIn(Graph.OptIn.SUITE_PROCESS_STANDARD)
 @Graph.OptIn(Graph.OptIn.SUITE_PROCESS_COMPUTER)
-@Graph.OptIn(Graph.OptIn.SUITE_PROCESS_PERFORMANCE)
```

Comment:
why is it ok to remove these annotations?

Reply:
http://tinkerpop.apache.org/docs/3.3.0/upgrade/#_graph_database_providers
> A number of test suites that were previously deprecated have been removed ...